weather_aplication/SearchViewController.swift

```diff
@@ -0,0 +1,121 @@
+import SnapKit
+import UIKit
+
+struct Location {
+    public var city: String?
+}
+
+class SearchViewController: UIViewController {
+    private let kSideMargin: CGFloat = 10
+    private let kTopMargin: CGFloat = 20
+    private let kBottomMargin: CGFloat = 15
```

Comment:
taka uwago-ciekawostka - przy używaniu SnapKita nie musisz używać `CGFloat`ów jako wartości constraintów, mogą to być też liczby typu `Int`, tak więc wystarczy wtedy pisać `private let kBottomMargin = 15` bez specyfikowania typu